lib/index.js

```diff
@@ -332,6 +351,25 @@ $('#login-form').submit(function(event) {
   authenticateUser(username, password);
 });
 
+// Event listener for register form
+$('#register-form').submit(function(event) {
+  $('#register-error').html('');
+
+  event.preventDefault();
+
+  let username = $('#user-register').val();
+  let password = $('#pass-register').val();
+  let passwordConfirmation = $('#confirm-pass-register').val();
+
+  if ((password === '') || (passwordConfirmation === '') || (username === '')) {
```

Comment:
I think this could be handled by raw html by adding the require tag to the input

Reply:
Didn't know that existed, thanks